code_of_conduct.md

```diff
@@ -0,0 +1,51 @@
+# Wanderlust - GSSoC Contributors Code of Conduct
+
+In our project, we believe in creating an open and inclusive space for everyone. To ensure a respectful and positive community, follow these key guidelines:
+
```

Comment:
Lets not have this section, it is there in contributing guidelines. Let's go with what I had written, its fine to start with "You'll never get .. " it is intentional

Reply:
so there should be only https://app.eraser.io/workspace/wQGMaE4BqFTVZ0tigke3  this .

Reply:
Yes @officeneerajsaini